cumulusci.yml

```diff
@@ -81,7 +81,13 @@ tasks:
         description: Adds picklist values to the given field, if they don't already exist.
         class_path: tasks.add_picklist_values.AddPicklistValues
         group: 'K-12: Metadata'
-        
+        options:
+            sobject: hed__Application__c
+            field: hed__Application_Type__c
+            values: "Elementary,Middle School,High School"
+            recordTypes: picklist
+            sorted: True
```

Comment:
I think moving the options into their own task makes sense, but I think it shouldn't replace `add_picklist_values`, but a separate task with a descriptive name (`add_application_types` or `add_application_type_picklist_values`) and a description that, er... describes its purpose

Reply:
I second this. Since K12 will be doing more of this in the future, the tasks should be specific.

Reply:
looking back I think I misread this are you suggesting the modularization as follows: a task for the options and a task for `add_application_type_picklist_values`

Reply:
deferring to @jstvz on the "options" question, but yes on the add_application_type_picklist_values (but I see you've done that, so thanks).

Reply:
To clarify:

1. Revert the `add_picklist_values` entry from `master`.
2. Create a new task entry, `add_application_type_picklist_values`, with the above options (`sobject`,`field`, etc).

You'll end up with two tasks, `add_picklist_values` and `add_application_type_picklist_values`.

Reply:
@jofsky @jstvz one more change that we didn't catch. Can we change "Elementary" to "Elementary School"?

Reply:
Thanks @jofsky for the update to the Elementary School picklist value. @jstvz just one more question. what does the "add_picklist_values" task do in this case? Is that just so we can run something completely custom from the cci command line? (e.g cci task run add_picklist_values <arbitrary stuffs here>?

Reply:
That's correct, @lifewithryan. While it isn't referenced in YAML, it's available via the CLI.